src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,11 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = (props) => {
```

Comment:
you can use destructuring here `({user})` or on the next line like `const {email, name} = props.user`

Reply:
Not fixed properly